packages/editor/src/components/parts/program/editor/EditorPart.tsx

```diff
@@ -0,0 +1,77 @@
+import { EditorData, Label, Script, Text, Widget } from '@axonivy/inscription-protocol';
+import { PartProps, usePartDirty, usePartState } from '../../../editors';
+import { useEditorContext, useMeta, useValidations } from '../../../../context';
+import { useEditorData } from './useEditorData';
+import { Input, MessageText, ScriptInput } from '../../../../components/widgets';
+
+export function useEditorPart(): PartProps {
+  const { config, defaultConfig, initConfig, reset } = useEditorData();
+  const compareData = (data: EditorData) => [data.userConfig, data.link];
+  const validation = useValidations(['javaClass']);
+  const state = usePartState(compareData(defaultConfig), compareData(config), validation);
+  const dirty = usePartDirty(compareData(initConfig), compareData(config));
+  return {
+    name: 'Editor',
+    state,
+    reset: { dirty, action: () => reset() },
+    content: <EditorPart />
+  };
+}
+
+const EditorPart = () => {
+  const { config, update } = useEditorData();
+  const { context } = useEditorContext();
+  const editorItems = useMeta('meta/program/editor', { context, type: config.javaClass }, []);
+
+  function isLabel(object: Widget): object is Label {
+    return 'text' in object;
+  }
+
+  function isScript(object: Widget): object is Script {
+    return 'requiredType' in object;
+  }
+
+  function isText(object: Widget): object is Text {
+    return !isLabel(object) && !isScript(object);
+  }
+
+  const renderWidgetComponent = (widget: Widget) => {
+    if (isLabel(widget)) {
+      const message = widget.text;
+
+      if (widget.multiline) {
+        // Split the text using a regular expression that considers periods followed by spaces or numbers
+        // and capital Words longer than 8 letters. But not when it's followed by space and a number
+        const sentences = message.split(/(?<=\.)\s+(?=\D|(?<!\.\s)\d)|\b(?=[A-Z]{8,}\b)/);
+        return (
+          <>
+            {sentences.map((sentence, index) => (
+              <div key={index}>
+                <MessageText message={{ severity: 'INFO', message: sentence.trim() }} />
+              </div>
+            ))}
+          </>
+        );
+      } else {
+        return <MessageText message={{ severity: 'INFO', message: message }} />;
+      }
+    } else if (isScript(widget)) {

```

Comment:
if you do a `return` in an `if` statement you can simply do another `if` instead of an `else if`.
Because it sometimes not a good coding style to have an `else if` statement without a closing final else.